Makefile

```diff
@@ -9,7 +9,7 @@ export PATH               := .bin:${PATH}
 export PWD                := $(shell pwd)
 export BUILD_DATE         := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 export VCS_REF            := $(shell git rev-parse HEAD)
-export QUICKSTART_OPTIONS ?= ""
+export QUICKSTART_OPTIONS ?=
```

Comment:
```suggestion
export QUICKSTART_OPTIONS ?= ""
```

Reply:
this change is because that raising an issue if this variable is empty

Reply:
Do I have your ok for this ?

Reply:
Sorry, not sure if I understand :) Does this mean the makefile will fail if QUICKSTART_OPTIONS is unset?

Reply:
yes it does, at least on mac

Reply:
OK, then please revert it because the makefile is often called without these options available, for example:

```
make test
```

Thanks!